test/controllers/MoreInformationControllerSpec.scala

```diff
@@ -151,6 +158,7 @@ class MoreInformationControllerSpec extends ControllerSpecBase {
         override val userAnswers: Option[UserAnswers] =
           Some(UserAnswers("some-cred-id")
             .set(CheckModePage, false).success.value
+            .set(NumberOfEntriesPage, OneEntry).success.value
           )
         lazy val result: Option[Call] = controller.backLink()
```

Comment:
Also need a test for back link pointing to the BulkFileUpload controller so that when that is implemented the test just needs controller name updated.